src/components/UpcomingEventCard.vue

```diff
@@ -174,15 +174,20 @@ export default {
 			const today = new Date();
 			const tomorrow = getDateTomorrow();
 			if (isDateSame(this.dateTimeStart, today)) {
-				return moment(this.dateTimeStart).format("HH:mm");
+				return dayjs(this.dateTimeStart).format("HH:mm");
 			} else if (isDateSame(this.dateTimeStart, tomorrow)) {
 				return "Tomorrow";
 			} else {
-				return moment(this.dateTimeStart).format("YYYY-MM-DD");
+				return dayjs(this.dateTimeStart).format("YYYY-MM-DD");
 			}
 		},
 	},
 	methods: {
+		talkUrlPruned(str) {
```

Comment:
The last little thing regarding this is to first check if there is a Talk link and then show that as the meeting link. Because if you have 2 links in the description and the last one is the talk link, then the other link gets chosen as the meeting link and the talk link disappears.

We can also talk with product first to hear how exactly they want the logic of this to work. I'll leave that up to you, if we need to talk with product first, we can merge it in the meantime and update the behaviour afterwards.